src/content/reference/okteto-cli.mdx

```diff
@@ -268,17 +269,27 @@ If you need to destroy external resources (like s3 buckets or other Cloud resour
 $ okteto destroy
 ```
 
+The `okteto destroy` command will only destroy resources matching the specified environment name. In a monorepo scenario, where you might have several manifests, you need to specify the correct manifest to destroy the specific development environment. This ensures that the appropriate destroy commands, as defined in your Okteto manifest, are executed within a particular scope.
+
+This way, executing the `okteto destroy` command on one environment will not delete the other. For example, if you have two development environments, `dev1` and `dev2`, and you run `okteto destroy` for dev1, it will only destroy the resources for dev1 and leave dev2 intact.
```

Comment:
```suggestion
This way, executing the `okteto destroy` command on one development environment will not delete the other. For example, if you have two development environments, `dev1` and `dev2`, and you run `okteto destroy` for dev1, it will only destroy the resources for dev1 and leave dev2 intact.
```